src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs

```diff
@@ -877,10 +877,25 @@ private static void SqlBulkCopyDataTable_Type(object paramValue, string expected
                     DropTable(conn, bulkCopyTableName);
                     xsql(conn, string.Format("create table {0} (f1 {1})", bulkCopyTableName, expectedBaseTypeName));
 
+                    string value = string.Empty;
```

Comment:
I understand what you are trying to do here, but this change will result in the test just exercising the string path in the bulk copy driver code instead of the date paths, reducing code coverage.